### ISSUE TITLE
How to handle .paginate metadata?

### ISSUE THREAD

**Issue by hlozancic** [rhwilr/adonis-bumblebee#6]
When transforming, for example, users collection and using .paginate to query them... Is it possible to get classic adonis response containing paginate metadata?

**When using:**
```
const users = await User.query().paginate()
return transform.collection(users, UserTransformer)

```
**Expected JSON response would be:**
```
{
	"data": {
		"total": 112,
		"perPage": 10,
		"page": 1,
		"lastPage": 12,
		"data": [{..... transformed users here .... } ....
...
}
```

**Comment by rhwilr** [comment]
Pagination and support for different serializers is planed for the next release.

**Comment by hlozancic** [comment]
Great, I'll try to jump and help if possible...

**Comment by rhwilr** [comment]
Pagination and metadata is now available in v1.2.0.
